lib/travis/build/script/julia.rb

```diff
@@ -11,10 +11,6 @@ module Travis
   module Build
     class Script
       class Julia < Script
-        DEFAULTS = {
-          julia: 'release',
```

Comment:
make this 1.0 ? I don't think all that many people do `language: julia` without any `julia:` specifiers, but may as well keep that possible?